quora-service/src/main/java/com/upgrad/quora/service/entity/AnswerEntity.java

```diff
@@ -15,7 +15,8 @@
 @Entity
 @Table(name = "answer")
 @NamedQueries({
-        @NamedQuery(name = "getAnswerById", query = "select a from AnswerEntity a where a.uuid=:uuid")
+        @NamedQuery(name = "getAnswerById", query = "select a from AnswerEntity a where a.uuid=:uuid"),
+        @NamedQuery(name = "getAllAnswersToQuestion", query = "select a from AnswerEntity a")
```

Comment:
You are basically getting all the answers irrespective of question. 
Please add where clause to complete this.

Reply:
ok